open_spiel/games/chess/chess_board.cc

```diff
@@ -1304,17 +1299,16 @@ void ChessBoard<kBoardSize>::GenerateKnightDestinations_(
 }
 
 // Pawn moves without captures.
-template <uint32_t kBoardSize>
 template <typename YieldFn>
-void ChessBoard<kBoardSize>::GeneratePawnDestinations_(
+void ChessBoard::GeneratePawnDestinations_(
     Square sq, Color color, const YieldFn &yield) const {
   int8_t y_direction = color == Color::kWhite ? 1 : -1;
   Square dest = sq + Offset{0, y_direction};
   if (InBoardArea(dest) && IsEmpty(dest)) {
     yield(dest);
 
-    // Test for double move.
-    if (IsPawnStartingRank(sq, color)) {
+    // Test for double move. Only defined on standard board
```

Comment:
`>= 5` ?

Reply:
I think what I meant is that this condition should be generalized to other board sizes.